### PR TITLE
Anuragjs,c

### PR DESCRIPTION
**Pull request by Anurag-ak99**
print Hello Hacktoberfest 2020! through Javascript

### CODE REVIEW

**Comment by Anurag-ak99** [comment]
please check and revert if any error occurs. Thanks

File: C#/hello.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace HelloWorld
+{
+  class Program
+  {
+    static void Main(string[] args)
+    {
+      Console.WriteLine("Hello Hactoberfest 2020!");
```
Comment (thiagoyeds) [inline C#/hello.cs]: Add "My name is Anurag-ak99!" the phrase, getting "Hello Hactoberfest 2020! My name is Anurag-ak99!"

File: C#/hello.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace HelloWorld
```
Comment (thiagoyeds) [inline C#/hello.cs]: Change namespace to Hello

File: javascript/anuraghello.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+// There are three ways to print in js
+console.log('Hello, Hacktoberfest2020!');
```
Comment (thiagoyeds) [inline javascript/anuraghello.js]: Change log to 'Hello Hactoberfest 2020! My name is Anurag-ak99!'

File: javascript/hellohack.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+// There are three ways to print in js
+document.write('Hello, World!');
+
+// document.write() is used when you want to print the content to the HTML document.
```
Comment (thiagoyeds) [inline javascript/hellohack.js]: Delete this file, just keep **hello-Anurag-ak99.js** and **hello-Anurag-ak99.cs**

File: javascript/anuraghello.js [hunk 5]
```diff
@@ -0,0 +1,2 @@
+// There are three ways to print in js
```
Comment (thiagoyeds) [inline javascript/anuraghello.js]: Change filename to **hello-Anurag-ak99.js**

File: C#/hello.cs [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+{
```
Comment (thiagoyeds) [inline C#/hello.cs]: Change filename to **hello-Anurag-ak99.cs**

**Review by thiagoyeds — CHANGES_REQUESTED** [review]
See the change requests and make them, and then I will accept your PR

File: C#/hello.cs (hunk 1 above)
Comment (Anurag-ak99) [reply C#/hello.cs]: okay

File: C#/hello.cs (hunk 6 above)
Comment (Anurag-ak99) [reply C#/hello.cs]: I have made all the changes. if everything is okay, then please accept my Pr

**Review by thiagoyeds — APPROVED** [review]
lgtm

**Pull request merged**